crates/sui-framework/packages/move-stdlib/tests/option_tests.move

```diff
@@ -289,3 +289,14 @@ fun destroy_or_no_drop() {
     let NoDrop {} = some;
     let NoDrop {} = none;
 }
+
+#[test]
+fun extract_or() {
+    let mut none = option::none<u64>();
```

Comment:
This is nice. Could we update the test for `destroy_or` to show the original option is not modified (i.e. it would have been copied if able)